antora.yml

```diff
@@ -8,3 +8,9 @@ nav:
 
 asciidoc:
   attributes:
+    minimum-php-version: 7.3
```

Comment:
10.8 supports PHP 7.2 - I guess you will do a separate PR to update the specific values?

Reply:
yes, this PR should just move the attribues as they are.
the next PR is 10.8 (resp 10.7 only) and contains the correct values